sormas-api/src/main/resources/strings.properties

```diff
@@ -613,6 +615,7 @@ infoDownloadImportTemplate = You can use this template .csv file to bring your d
 infoDownloadErrorReport = If there were any rows that could not be imported, you will be offered a .csv file containing all these rows as well as the error descriptions.
 infoDownloadImportGuide = If this is your first time importing data into SORMAS, we strongly recommend to read the import guide first.
 infoEventParticipantAlreadyExisting = For this person, an event participant already exists in this event. You can either continue with this event participant or go back to the event selection.
+infoExpectedFollowUpUntilDate = The expected follow-up until date %s is based on its %s (%s)
```

Comment:
I'd prefer to use dedicated strings for contacts and cases; injecting other strings into existing strings like this is always a bit risky because there might be languages for which this makes translation pretty hard